pkg/sti/api/script.go

```diff
@@ -0,0 +1,20 @@
+package api
+
+// Script defines script names used by STI process.
+type Script string
```

Comment:
I don't think this will improve readability and personally I don't like masking a 'string' to 'Script' type as it makes me search how the 'Script' type is defined. I would rather prefer to have generic string and then have constants for the script names (which seems more idiomatic to me). @smarterclayton thoughts?